repose-aggregator/src/docs/asciidoc/filters/valkyrie-authorization.adoc

```diff
@@ -117,7 +117,7 @@ For more information about specific response codes that *Repose* will receive fr
 ====
 
 == Account Admins
-If a user has the `account_admin` role in Valkyrie, when `enable-bypass-account-admin` is configured to be `true` (the default is `false`), the Valkyrie filter will pass the request along regardless of whether or not the device permission check fails.
+If a user has the `account_admin` role in Valkyrie, when `enable-bypass-account-admin` is configured to be `true` (Default: `false`), the Valkyrie filter will pass the request along regardless of whether or not the device permission check fails.
```

Comment:
I wonder if we want to keep the default noted on this line. I noticed it did not follow the new line pattern that all the others do, and that is because it is an inline aside, and so it can't really. That made me question what could be done about it, and I landed on removing it since leaving it in means that we have multiple sources of truth for the default value of that attribute. 

I'm actually fine leaving it as is, but have a slight preference toward removing it.

Reply:
Removed since it is stated as the `Default` in the callout a few lines later.